### PR TITLE
Mbo/us/aaicddc 19 setup workflow

### PR DESCRIPTION
**Pull request by ajithroy-iqvia**
- Configured workflow module
- Replaced Edit draft view with new onw
- Permissions set correctly for the contributors and managers

### CODE REVIEW

File: codebase/config/sync/views.view.study_administration.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 uuid: 0b3fd574-ca20-42bf-80ba-36c4d92d73df
 langcode: en
-status: true
+status: false
```
Comment (ataperekiq) [inline codebase/config/sync/views.view.study_administration.yml:3]: what does this do?

File: codebase/config/sync/views.view.draft_content.yml [hunk 2]
```diff
@@ -0,0 +1,815 @@
+uuid: ad739a60-6490-4cda-973f-ddaf3d18020d
```
Comment (ataperekiq) [inline codebase/config/sync/views.view.draft_content.yml:1]: Does this mean the other view for Manage Draft Studies can be deleted?

File: codebase/config/sync/group.role.study-contributor.yml [hunk 3]
```diff
@@ -16,7 +16,9 @@ permissions:
   - 'access group_node overview'
   - 'update any group_node:study entity'
   - 'update own group_node:study entity'
+  - 'view all group_node:study entity revisions'
```
Comment (ataperekiq) [inline codebase/config/sync/group.role.study-contributor.yml:19]: does this give a contributor access to revisions of other studies they should not see?

File: codebase/config/sync/views.view.study_administration.yml (hunk 1 above)
Comment (ajithroy-iqvia) [reply codebase/config/sync/views.view.study_administration.yml:3]: This change will disable the view and keep it in the disabled views section.

File: codebase/config/sync/views.view.draft_content.yml (hunk 2 above)
Comment (ajithroy-iqvia) [reply codebase/config/sync/views.view.draft_content.yml:1]: I have kept it undeleted but in the disabled views section.

File: codebase/config/sync/group.role.study-contributor.yml (hunk 3 above)
Comment (ajithroy-iqvia) [reply codebase/config/sync/group.role.study-contributor.yml:19]: I noted that naming of the permission in the yml file but in the front-end, I have enabled only the following two permissions- 

- Revisions: View full version history
- Revisions: View specific entity revisions

Comment (ajithroy-iqvia) [reply codebase/config/sync/group.role.study-contributor.yml:19]: I have checked if contributor can access other group studies/revisions and they can't.

**Review by ataperekiq — APPROVED** [review]

**Pull request merged**